uint8 set_homepoint=1
uint8 pause=2
uint8 unpause=3

uint8 action
